apps/accounts/views.py

```diff
@@ -12,24 +13,37 @@
 from accounts.serializers import UserProfileSerializer
 
 
+def find_user(identity):
+    try:
+        return UserProfile.objects.get(
+            Q(fxa_id=identity['uid']) | Q(email=identity['email']))
+    except UserProfile.DoesNotExist:
+        return None
+    except UserProfile.MultipleObjectsReturned:
+        # This shouldn't happen, so let it raise.
+        raise
```

Comment:
What's the point of re-raising here, without doing anything else? Is it to add the comment? Or did you mean to add some logging or something?